types/signed_header.go

```diff
@@ -93,6 +92,31 @@ var (
 	ErrSignatureVerificationFailed = errors.New("signature verification failed")
 )
 
+// VerifyCentralizedSequencer checks if a SignedHeader came from the expected centralized sequencer, given a genesis file.
+func (sh *SignedHeader) VerifyCentralizedSequencer(genesis *cmtypes.GenesisDoc) error {
+	if len(genesis.Validators) != 1 {
```

Comment:
no need to check this every time, its already done once in the beginning: https://github.com/rollkit/rollkit/blob/main/types/state.go#L63 ?